openrtb_ext/device.go

```diff
@@ -70,8 +70,8 @@ type ExtDevicePrebid struct {
 
 // ExtDeviceInt defines the contract for bidrequest.device.ext.prebid.interstitial
 type ExtDeviceInt struct {
-	MinWidthPerc  uint64 `json:"minwidtheperc"`
-	MinHeightPerc uint64 `json:"minheightperc"`
+	MinWidthPerc  int64 `json:"minwidtheperc"`
```

Comment:
The change to the new openrtb library made the related ints int signed ints, so doing this here rather than having a type conversion later when the generated widths/heights need to be put into the openrtb structures.